caster/apps/gitbash.py

```diff
@@ -30,86 +28,86 @@ class GitBashRule(MergeRule):
     pronunciation = "git bash"
 
     mapping = {
-        "initialize repository":
-            Text("git init") + Key("enter"),
-        "add":
-            R(Key("g, i, t, space, a, d, d, space, dot, enter"),
+        "git|get base":
```

Comment:
I believe that the grammar parser turns this into `git` and `get base` instead of `git base` and `get base` as intended. I recommend using parentheses for explicit grouping, such as `(git | get) base`.

Other than that, I love all of these changes.

Reply:
I think I fixed these all up now